docs/docs/02-Installation/03-archlinux.md

```diff
@@ -43,6 +43,48 @@
 
 - `karakeep-web.service`: Provide karakeep webui service, uses `3000` port by default.
 
-- `karakeep-works.service`: Provide karakeep works service, no port.
+- `karakeep-workers.service`: Provide karakeep workers service, no port.
 
 - `karakeep-browser.service`: Provide browser headless service, uses `9222` port by default.
+
+Now `karakeep` depends on `meilisearch`, and `karakeep-workers.service` wants `meilisearch.service`, starting `karakeep.target` will start `meilisearch.service` at the same time.
+
+## How to Migrate from Hoarder to Karakeep
+
+The PKGBUILD has been fully updated to replace all references to `hoarder` with `karakeep`. If you want to preserve your existing `hoarder` data during the upgrade, please follow the steps below:
+
+**1. Stop the old services**
+
+```shell
+sudo systemctl stop hoarder-web.service hoarder-worker.service hoarder-browser.service
+sudo systemctl disable --now hoarder.target
+```
+
+**2. Uninstall Hoarder**  
+After uninstalling, you can manually remove the old `hoarder` user and group if needed.
+```shell
+paru -R hoarder
+```
+
+**3. Rename the old data directory**
+```shell
+sudo mv /var/lib/hoarder /var/lib/karakeep
+```
```

Comment:
Shouldn't we ask people to move their env file as well? Otherwise they might lose their configuration?